Casks/q/qqbrowser.rb

```diff
@@ -0,0 +1,33 @@
+cask "qqbrowser" do
+  arch arm: "m1", intel: "intel"
+
+  version "5.0.4.213"
+  sha256 arm:   "d3524eba8b93a111d93980a37366ad9a43263c1404fb9949eafd1658f927a511",
+         intel: "d8b997236ae96da16bf07bfbecf0b62096be018871513f1b16c068963f8b2339"
+
+  url "https://pcchannel.imtt.qq.com/channel/QQBrowser_#{arch}.dmg"
+  name "QQBrowser"
+  name "QQ浏览器"
+  desc "QQ web browser"
+  homepage "https://browser.qq.com/mac/"
+
+  # The appcast file is outdated and does not contain the latest version
+  # https://dldir1v6.qq.com/invc/tt/QB/Public/qbmac/appcast.xml
+  livecheck do
+    skip "No reliable way to get version info"
+  end
```

Comment:
```suggestion
  livecheck do
    url :url
    strategy :extract_plist do |items|
      items["com.tencent.qqbrowserappmac"].short_version
    end
  end
```

<https://docs.brew.sh/Brew-Livecheck#extractplist-strategy-block>